### PR TITLE
lab: kube: add flag --version to create subcommand

### LINKED ISSUE #111: exo lab kube: add support for user-specified k8s version
Sometimes running `exo lab kube create` fails because the latest version of Kubernetes (retrieved at https://dl.k8s.io/release/stable.txt) is not yet packaged by Ubuntu). For example right now  the latest k8s version advertised is `1.13.4`, however creating exokube instances fails because the latest packaged version of the *kubelet/kubeadm/kubectl* utilities is `1.13.3`.

Adding an optional `--version` flag to the `exo lab kube create` command would allow users to specify a Kubernetes version (e.g. `1.13.3`) as a temporary workaround.

### PR DESCRIPTION
**Pull request by falzm**
Fixes #111.

### CODE REVIEW

File: cmd/kube_create.go [hunk 1]
```diff
@@ -198,10 +198,15 @@ var kubeCreateCmd = &cobra.Command{
 			return err
 		}
 
-		kubernetesVersion, err := fetchKubernetesVersion()
+		kubernetesVersion, err := cmd.Flags().GetString("version")
```
Comment (pierre-emmanuelJ) [inline cmd/kube_create.go]: I think can be cool to check if asking version exist, example:
https://github.com/kubernetes/kubernetes/releases/tag/v1.13.4
https://github.com/kubernetes/kubernetes/releases/tag/v{version}
https://github.com/kubernetes/kubernetes/releases/tag/{version}

Comment (greut) [reply cmd/kube_create.go]: https://github.com/exoscale/cli/blob/742b3ae2000994b95557234591e038f9b0639b23/cmd/kube_create.go#L518-L544

Comment (falzm) [reply cmd/kube_create.go]: Thanks @pierre-emmanuelJ & @greut, will do this.

**Comment by falzm** [comment]
> LGTM if you have tested also it's good to me

Yep, tested OK 👍

**Comment by dajac** [comment]
LGTM

**Pull request merged**